src/components/Dashboard.vue

```diff
@@ -60,6 +60,7 @@ Vue.component('navbar', NavBar);
 Vue.component('onboarding', Onboarding);
 
 const firebaseConfig = require('@/firebaseConfig.js');
+const subjectColors = require('@/assets/subjects/subjects.json');
```

Comment:
Same with this line and its reference later in the file

Reply:
Same as last reply. Branched from subject colors